Furball.Engine/Engine/Graphics/Drawables/Primitives/CirclePrimitiveDrawable.cs

```diff
@@ -1,33 +1,33 @@
-using Furball.Engine.Engine.Graphics.Drawables.Managers;
-using Microsoft.Xna.Framework;
-using Xssp.MonoGame.Primitives2D;
-
-namespace Furball.Engine.Engine.Graphics.Drawables.Primitives {
-    /// <summary>
-    /// Simple Circle Drawable
-    /// </summary>
-    public class CirclePrimitiveDrawable : ManagedDrawable {
-        /// <summary>
-        ///     The detail of the circle
-        /// </summary>
-        public int Detail;
-        /// <summary>
-        /// Creates a Circle
-        /// </summary>
-        /// <param name="position">Where to Draw</param>
-        /// <param name="radius">How big should the Circle be</param>
-        /// <param name="outlineColor">What Color should the Border be</param>
-        /// <param name="detail">How many sides are on the circle</param>
-        public CirclePrimitiveDrawable(Vector2 position, float radius, Color outlineColor, int detail = 25) {
-            this.CircleRadius  = radius;
-            this.Circular      = true;
-            this.ColorOverride = outlineColor;
-            this.Position      = position;
-            this.Detail        = detail;
-        }
-
-        public override void Draw(GameTime time, DrawableBatch batch, DrawableManagerArgs args) {
-            batch.SpriteBatch.DrawCircle(args.Position * FurballGame.VerticalRatio, this.CircleRadius * FurballGame.VerticalRatio, this.Detail, args.Color, 0f);
-        }
-    }
-}
+//TODO(Eevee)@Vixie: circle drawing shader stuff thing ass
+//using Furball.Engine.Engine.Graphics.Drawables.Managers;
+//
+//namespace Furball.Engine.Engine.Graphics.Drawables.Primitives {
+//    /// <summary>
+//    /// Simple Circle Drawable
+//    /// </summary>
+//    public class CirclePrimitiveDrawable : ManagedDrawable {
+//        /// <summary>
+//        ///     The detail of the circle
+//        /// </summary>
+//        public int Detail;
+//        /// <summary>
+//        /// Creates a Circle
+//        /// </summary>
+//        /// <param name="position">Where to Draw</param>
+//        /// <param name="radius">How big should the Circle be</param>
+//        /// <param name="outlineColor">What Color should the Border be</param>
+//        /// <param name="detail">How many sides are on the circle</param>
+//        public CirclePrimitiveDrawable(Vector2 position, float radius, Color outlineColor, int detail = 25) {
+//            this.CircleRadius  = radius;
+//            this.Circular      = true;
+//            this.ColorOverride = outlineColor;
+//            this.Position      = position;
+//            this.Detail        = detail;
+//        }
+//
+//        public override void Draw(double time, DrawableBatch batch, DrawableManagerArgs args) {
+//            batch.SpriteBatch.DrawCircle(args.Position * FurballGame.VerticalRatio, this.CircleRadius * FurballGame.VerticalRatio, this.Detail, args.Color, 0f);
+//        }
+//    }
+//}
+//
```

Comment:
This cant be merged without this feature, this is a p important thing to be missing

Reply:
ive said it multiple times that im gonna maybe do this after the merge

Reply:
and all the times ive told you youve never had a problem with it

Reply:
> and all the times ive told you youve never had a problem with it

I dont recall this, and ive started using it more recently, so thats probably whats have to do with it

Reply:
im seriously getting fucking sick of this pr

Reply:
"yeah im fine using textures for circles"
and now all the sudden you need em because this pr exists

Reply:
> "yeah im fine using textures for circles"
> and now all the sudden you need em because this pr exists

I've needed variable outline widths since then, that's why i started using it

Reply:
Ill just copy my line based circle drawable from another project, now that i have the time to, i should just be able to extend CirclePrimitiveDrawable off of PathDrawable